commands/NotifyRandomOpepenSet.ts

```diff
@@ -24,12 +25,25 @@ export default class NotifyRandomOpepenSet extends BaseCommand {
       return
     }
 
-    const { default: SetSubmission } = await import('App/Models/SetSubmission')
+    const minResult = await SetSubmission.query()
+      .whereNotNull('setId')
+      .min('bot_featured_count as min_count')
```

Comment:
when we use model attributes throughout our codebase we normally _"camelCase"_ it: `botFeaturedCount` similar to how `setId` is _camelCased_ above. for this to work we have to define the property in the `SetSubmission` model.

the _"snake_casking"_ is only applied on the database level.